rafthttp/transport.go

```diff
@@ -202,11 +202,16 @@ func (t *Transport) Stop() {
 	if tr, ok := t.pipelineRt.(*http.Transport); ok {
 		tr.CloseIdleConnections()
 	}
+	t.peers = nil
+	t.remotes = nil
 }
 
 func (t *Transport) AddRemote(id types.ID, us []string) {
 	t.mu.Lock()
 	defer t.mu.Unlock()
+	if t.remotes == nil {
+		panic("transport stopped")
```

Comment:
Is there a way to cleanly shutdown a http server right now? closing the listener does not mean closing all the existing connections.